extensions/2.0/Khronos/KHR_lights_environment/README.md

```diff
@@ -13,14 +13,23 @@
 
 Draft
 
+Open issues:
+Should Spherical Harmonics be supported or should clients be forced to evaluate the specular cubemap for diffuse contribution.  
+The question comes down to if resolution supplied by 9 coefficients SH is enough for most usecases?
+Since the SH are optional, and I believe the quality using 9 coefficients is reasonable,  I am leaning towards keeping them.   
+
+If compressed texture format is used how should pre-filtering for roughness reflection be done?
```

Comment:
The compressed texture can be ready by the filtering code directly to create the mip chain of roughness, etc.  Remember that the three.js PMREM reads from one texture to create the pre-filtered results.  If the compressed format is supported, it will just work.  If it isn't supported one can transcode to a supported format.  There is no cost to using a compressed format when creating the filtered mips.

Reply:
As I stated above - the cost is in inconsistency with regards to GPU memory usage.
As a content creator you have a hard time knowing what the GPU budget will end up as.

Reply:
I propose to make it optional to include the pre-filtered mips for compressed texture formats (bc6h).
This is solely for usecases where content creators want to have a better control and knowledge of target GPU memory.

@bhouston Is this a solution that is ok with you?